samples/sensor/hx711/prj.conf

```diff
@@ -0,0 +1,29 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_SENSOR=y
+CONFIG_HX711=y
+CONFIG_GPIO=y
+CONFIG_HX711_CALIBRATION_WEIGHT=500
+
+
+CONFIG_PRINTK=y
+
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=4
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_RTT_MODE_DROP=y
+CONFIG_LOG_MODE_NO_OVERFLOW=y
+CONFIG_LOG_PRINTK=y
+CONFIG_LOG_PRINTK_MAX_STRING_LENGTH=256
+CONFIG_LOG_BUFFER_SIZE=4096
+CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE=256
+CONFIG_LOG_STRDUP_BUF_COUNT=64
+CONFIG_LOG_STRDUP_MAX_STRING=64
+CONFIG_LOG_BACKEND_SHOW_COLOR=n
+CONFIG_LOG_BACKEND_FORMAT_TIMESTAMP=y
+CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
+
+CONFIG_CONSOLE=y
+CONFIG_USE_SEGGER_RTT=y
```

Comment:
The `RTT` entries might be better in a board-specific conf file in the boards folder, since they won't be available or meaningful defaults on every board, and only items common to every board should be placed in this file.

Reply:
Agreed. I think most of the LOG configurations should also be removed.

Reply:
They have been removed.
I do not know why they still show up here....